examples/graphs/proteins_pytorch_geometric/const.yaml

```diff
@@ -1,19 +1,21 @@
 name: proteins_pytorch_geometric
 hyperparameters:
-  global_batch_size: 20
+  global_batch_size: 60
   dataset: PROTEINS
-  lr: 5.e-4
-  weight_decay: 1.e-4
-  hidden_channels: 32
+  lr: 1.e-3
```

Comment:
Is there a reason this is different from the original?

Reply:
fixed.